libdnf/dnf-sack.cpp

```diff
@@ -2282,12 +2282,10 @@ void dnf_sack_filter_modules(DnfSack *sack, GPtrArray *repos)
         nevra.clear();
```

Comment:
Hmm. `nevra.getName().c_str()` - We get pointer to the internal array currently used by the string object to store the characters that conform its value. We push the pointer into a vector `names`. And then we invalidate nevra (clear) and reuse it. So all pointers in the vector points to the same nevra.